wicket/src/dispatch.rs

```diff
@@ -13,64 +13,51 @@ use slog::Drain;
 
 use crate::{upload::UploadArgs, Runner};
 
-#[derive(Debug, Parser)]
-#[command(version, author = "Oxide Computer Company")]
-pub struct WicketApp {
-    /// Login shell arguments.
-    ///
-    /// Wicket is designed to be a login shell for use over ssh. If no arguments are specified,
-    /// wicket behaves like a TUI. However, if arguments are specified with "-c" (as in other login
-    /// shells e.g. bash -c), wicketd accepts an upload command.
-    ///
-    /// Login shell arguments are provided in a quoted form, so we expect a single String here.
-    /// This string is split using shell quoting logic to get the actual arguments.
-    #[arg(short = 'c', allow_hyphen_values = true)]
-    shell_args: Option<String>,
+pub fn exec() -> Result<()> {
+    // TODO: make this configurable?
```

Comment:
wicketd's smf manifest has a property for its address; since wicket lives in the same zone, I believe it should be able to look it up. I can take that as a followup if you want, or you can crib from MGS, which [looks up its own SMF properties](https://github.com/oxidecomputer/omicron/blob/7c2e28c1f6a77ccab3a96deea2661846ee7f267e/gateway/src/bin/mgs.rs#L139).

Reply:
I'll try doing this in a followup.